oojs/The-keyword-this.md

```diff
@@ -1,21 +1,62 @@
 # The keyword _this_
 
-In most classical object oriented languages, there is a way to refer to the current object.  With JavaScript the keyword _this_ is used.  
+In most classical object oriented languages, there is a way to refer to the current object. With JavaScript the keyword `this` is used.
 
-Now _this_ maybe the most misunderstood aspect of the language.  One way to understand it better is to think of it as the parameter _this_, as it behaves almost exactly as a parameter with two exceptions. 
+Now `this` maybe the most misunderstood aspect of the language. One way to understand it better is to think of it as the parameter `this`, as it behaves almost exactly as a parameter with two exceptions.
 
-Firstly you, the developer, cannot pick a name, it will always be the parameter _this_.
+Firstly you, the developer, cannot pick its name - it will always be called `this`.
 
-Secondly, values are bound differently to _this_ compared with how values are bound to other parameters.
+Secondly, values are bound differently to `this` compared with how values are bound to other parameters.
 
-When you have a period . to the left of a function invocation i.e. it was looked up as a property of an object on the Left Hand Side, 
+When you invoke a method of an object (i.e. using dot notation), `this` will refer to the object on the left of the function invocation:
```

Comment:
I think this can be added to the explanation as this was also described in the udacity course